index.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import dotenv from 'dotenv';
+import chalk from 'chalk';
+import swaggerDoc from 'swagger-ui-express';
+import { swaggerControllers } from './controllers/index.js';
+
+dotenv.config();
+
+const app = express();
+
+const { PORT = 3000 } = process.env;
+
+app.use(morgan('short'));
+
+app.use(cors());
+
+app.use(express.json());
+
+app.use('/docs', swaggerDoc.serve, swaggerDoc.setup(swaggerControllers));
+console.log(
+  chalk.bgYellow.italic(`Swagger here ----> http://localhost:${PORT}/docs`)
+);
+
+app.use((_req, res) => {
+  res.status(404).json({
+    status: 'error',
+    message: 'Something went wrong...',
+  });
+});
+
+app.use((err, _req, res, _next) => {
+  const { status = 500 } = err;
+
+  res.status(status).json(err.message);
```

Comment:
log the whole error to have ability for debugging later on and send default text for unhandled error 'Something went wrong'

Reply:
Done.